src/main/java/de/retest/recheck/report/TestReportFilter.java

```diff
@@ -130,6 +138,13 @@ static IdentifyingAttributesDifference filter( final Element element,
 						: new IdentifyingAttributesDifference( element.getIdentifyingAttributes(), diffs ) ) );
 	}
 
+	static InsertedDeletedElementDifference filter( final InsertedDeletedElementDifference insertedDeletedDiff,
+			final Filter filter ) {
+		final Element insertedDeleted =
+				insertedDeletedDiff.isInserted() ? insertedDeletedDiff.getActual() : insertedDeletedDiff.getExpected();
```

Comment:
Should we move this into `InsertedDeletedElementDifference`. I assume, we need this a couple more times...